hnn_core/optimization.py

```diff
@@ -358,7 +358,7 @@ def _optrun(drive_params_updated, drive_params_static, net, tstop, dt,
 
     opt_params['optiter'] += 1
 
-    return avg_rmse, avg_rmse_unweighted  # nlopt expects error
```

Comment:
This shouldn't affect the ability to access the RMSE at each step should the user select `return_rmse=True`. Note that the RMSE for this step is stored in the `opt_params['iter_avg_rmse']` list that gets modified within the current function.

Reply:
Looks good. Can you also update the docstring?

Reply:
Done! Also just update whats_new.rst. Not that it'll be very relevant in a few days :)